scalar.go

```diff
@@ -74,3 +75,29 @@ func IsID(value string) bool {
 	}
 	return strings.HasPrefix(string(decoded), "gid://")
 }
+
+// NullableValue can be used to unset a value using an OpsLevel input struct type, should always be instantiated using a constructor.
+type NullableValue[T any] struct {
+	Value   T
+	SetNull bool
+}
```

Comment:
Is there a reason we are not doing

```suggestion
type NullableValue[T any] struct {
	sql.Null[T]
}
```